### ISSUE TITLE
layout.mapbox.layers.below attribute vs. Plotly.react performance

### ISSUE THREAD

**Issue by platinops** [plotly/plotly.js#6252]
I'm encountering issues with the performance of `Plotly.react` (or `Plotly.addTraces`) when used on a graph with multiple mapbox raster layers that are all plotted below a scattermapbox layer.

The following codepen illustrates the issue: https://codepen.io/platinops/pen/rNdxpOz

- Case 1 adds 10 traces on a mapbox graph with 1 raster layer => no issues.
- Case 2 adds 10 traces on a mapbox graph with 2 raster layers, both with [mapbox.layers.below](https://plotly.com/javascript/reference/layout/mapbox/#layout-mapbox-layers-items-layer-below)='traces'. Mapbox layers are properly ordered. When the 10 traces are added, browser devtools show that the tiles of both layers are downloaded several times for no apparent reason. This leads to increase CPU usage and decreased performance of my app.
- Case 3 is the same as the previous one, but this time I referred the drawing order of one raster layer to the order. On the plus side, the tiles are not downloaded again upon adding new traces. The layers are however no longer plotted in the correct order. Plotly throws a warning "Trying to add layer with *below* value rail referencing a layer that does not exist or that does not yet exist.".

Are case 2 and/or case 3 a bug?

**Comment by gvwilson** [comment]
Hi - we are trying to tidy up the stale issues and PRs in Plotly's public repositories so that we can focus on things that are still important to our community. Since this one has been sitting for several years, I'm going to close it; if it is still a concern, please add a comment letting us know what recent version of our software you've checked it with so that I can reopen it and add it to our backlog. Thanks for your help - @gvwilson

**Comment by platinops** [comment]
Hello @gvwilson, I'm still having this issue. I created a new pen with the latest version of Plotly.JS (3.3.1): https://codepen.io/platinops/pen/azZprNz

The [layout-map-layers-items-layer-below](https://plotly.com/python/reference/layout/#layout-map-layers-items-layer-below) field mentions:

> Determines if the layer will be inserted before the layer with the specified ID. If omitted or set to '', the layer will be inserted above every existing layer.

What is meant by "the specified ID" though? What is the specified ID for tile layers I added myself? How can I determine the drawing sequence of tile layers?

In the pen's case, only the second case manages to get the drawing order correct. But this case has the issue that any modification to the traces causes the tiles of the underlying layers to be re-downloaded.

<img width="1033" height="524" alt="Image" src="https://github.com/user-attachments/assets/c90f7c83-67d7-4dcb-b9ad-96dec33ea274" />
